cmd/plasmacli/subcmd/eth/query/balance.go

```diff
@@ -32,6 +32,8 @@ var balanceCmd = &cobra.Command{
 			addr = ethcmn.HexToAddress(args[0])
 		}
 
+		cmd.SilenceUsage = true
+
 		balance, err := plasmaContract.BalanceOf(nil, addr)
```

Comment:
Unrelated but might as well ->Ca n the if statement be pushed up here. Odd extra line before